UserList-LIUCHEN/UserList-LIUCHEN/Controller/HomeViewController.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  HomeViewController.swift
+//  UserList-LIUCHEN
+//
+//  Created by LC on 2020/6/23.
+//  Copyright © 2020 LC. All rights reserved.
+//
+
+import UIKit
+
+let apiManager = APIManager()
```

Comment:
Why are you declaring ApiManager in global scope and why in a this viewcontroller?

Reply:
In global scope  the apiManager is lazy initialize also is singleton pattern but should be put it in Applegate not in the viewcontroller